.github/workflows/reusable-lightweight.yml

```diff
@@ -45,102 +51,151 @@ permissions:
   packages: read
 
 concurrency:
-  group:  ${{ github.head_ref || github.run_id }}
+  # allows for running this workflow simultaneously with main `resable-build-test-release.yml
+  group:  ${{ github.head_ref || github.run_id }}-lightweight
   cancel-in-progress: true
 
 jobs:
+  get-called-ref:
+    name: Get called ref
+    runs-on: ubuntu-latest
+    outputs:
+      ref: ${{ steps.get-ref.outputs.caller-ref }}
+    steps:
+      - id: get-ref
+        uses: splunk/addonfactory-workflow-addon-release/.github/actions/_called-wf-ref@fix/lightweight-workflow-ADDON-66448
```

Comment:
Please note that this PR can't be merged until this line is changed to `uses: splunk/addonfactory-workflow-addon-release/.github/actions/_called-wf-ref@main`. This is the only action that is supposed to be always called from main.